samples/cpp/benchmark/throughput_benchmark/main.cpp

```diff
@@ -22,8 +22,12 @@ int main(int argc, char* argv[]) {
     try {
         slog::info << "OpenVINO:" << slog::endl;
         slog::info << ov::get_openvino_version();
-        if (argc != 2) {
-            slog::info << "Usage : " << argv[0] << " <path_to_model>" << slog::endl;
+
+        std::string device_name = "CPU";
+        if (argc == 3) {
+            device_name = TSTRING2STRING(argv[2]);
```

Comment:
```suggestion
            device_name = argv[2];
```
This main doesn't take tchars